kroxylicious-operator/pom.xml

```diff
@@ -158,12 +158,12 @@
         <dependency>
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
-            <scope>provided</scope>
+            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <scope>compile</scope>
+            <scope>provided</scope>
```

Comment:
are you able to run `OperatorMain` from intellij with these changes? It fails for me with:

`Exception in thread "main" java.lang.NoClassDefFoundError: com/fasterxml/jackson/core/JsonProcessingException`

and if I set jackson-core back to `compile` it then fails on a jackson YAML class not being found

Reply:
Try now

Reply:
What file has a compile error for you in IDEA? I don't seem to have that problem.

Reply:
Not a compile error, runtime error when I run OperatorMain to run the operator from IDE

Reply:
Fixed

Reply:
thanks, yeah I've had a further play with maven just to ensure I'm not imagining this:

https://github.com/robobario/transitive-sadness

TL:DR; setting a transitive compile dependency to test/provided is a pain for tools because maven then thinks `kroxylicious-operator` runtime scope should not include those deps. This would be fine for a library where we don't have a main method we are interested in running from IDE.